projects/1.20-fabric/assets/rogues-and-warriors/rogues/lang/zh_cn.json

```
@@ -0,0 +1,112 @@
+{
+  "itemGroup.rogues.general": "盗贼与战士",
+
+  "item.rogues.rogue_spell_book": "盗贼手册",
+  "item.rogues.warrior_spell_book": "战士秘籍",
+
+  "item.rogues.flint_dagger": "燧石匕首",
+  "item.rogues.golden_dagger": "金匕首",
+  "item.rogues.iron_dagger": "铁匕首",
+  "item.rogues.diamond_dagger": "钻石匕首",
+  "item.rogues.netherite_dagger": "下界合金匕首",
+  "item.rogues.aeternium_dagger": "太古合金匕首",
+  "item.rogues.ruby_dagger": "红宝石匕首",
+
+  "item.rogues.iron_sickle": "铁镰刀",
+  "item.rogues.golden_sickle": "金镰刀",
+  "item.rogues.diamond_sickle": "钻石镰刀",
+  "item.rogues.netherite_sickle": "下界合金镰刀",
+  "item.rogues.aeternium_sickle": "太古合金镰刀",
+  "item.rogues.ruby_sickle": "红宝石镰刀",
+
+  "item.rogues.stone_double_axe": "石双刃斧",
+  "item.rogues.iron_double_axe": "铁双刃斧",
+  "item.rogues.golden_double_axe": "金双刃斧",
+  "item.rogues.diamond_double_axe": "钻石双刃斧",
+  "item.rogues.netherite_double_axe": "下界合金双刃斧",
+  "item.rogues.aeternium_double_axe": "太古合金双刃斧",
+  "item.rogues.ruby_double_axe": "红宝石双刃斧",
+
+  "item.rogues.iron_glaive": "铁长柄刀",
+  "item.rogues.golden_glaive": "金长柄刀",
+  "item.rogues.diamond_glaive": "钻石长柄刀",
+  "item.rogues.netherite_glaive": "下界合金长柄刀",
+  "item.rogues.aeternium_glaive": "太古合金长柄刀",
+  "item.rogues.ruby_glaive": "红宝石长柄刀",
+
+  "item.rogues.rogue_armor_head": "盗贼面罩",
+  "item.rogues.rogue_armor_chest": "盗贼外套",
+  "item.rogues.rogue_armor_legs": "盗贼裤子",
+  "item.rogues.rogue_armor_feet": "盗贼靴子",
+  "item.rogues.assassin_armor_head": "刺客头罩",
+  "item.rogues.assassin_armor_chest": "刺客外套",
+  "item.rogues.assassin_armor_legs": "刺客裤子",
+  "item.rogues.assassin_armor_feet": "刺客靴子",
+
+  "item.rogues.warrior_armor_head": "战士头盔",
+  "item.rogues.warrior_armor_chest": "战士胸甲",
+  "item.rogues.warrior_armor_legs": "战士护腿",
+  "item.rogues.warrior_armor_feet": "战士靴子",
+  "item.rogues.berserker_armor_head": "狂战士头盔",
+  "item.rogues.berserker_armor_chest": "狂战士胸甲",
+  "item.rogues.berserker_armor_legs": "狂战士护腿",
+  "item.rogues.berserker_armor_feet": "狂战士靴子",
+
+  "entity.minecraft.villager.arms_merchant": "武具商人",
+  "entity.minecraft.villager.rogues:arms_merchant": "武具商人",
+  "block.rogues.arms_workbench": "武具工作台",
+  "block.rogues.arms_workbench.hint": "武具商人村民的工作台。",
+
+  "spell.rogues.slice_and_dice.name": "切割",
+  "spell.rogues.slice_and_dice.description": "进入战斗姿态，期间每次攻击给予伤害加成，可堆叠{max_stack}层。持续{effect_duration}秒。",
+  "effect.rogues.slice_and_dice": "切割",
+  "effect.rogues.slice_and_dice.description": "攻击伤害提升",
+
+  "spell.rogues.shock_powder.name": "眩晕粉尘",
+  "spell.rogues.shock_powder.description": "令周围的敌人眩晕{effect_duration}秒。",
+  "effect.rogues.shock": "眩晕",
+  "effect.rogues.shock.description": "无法行动",
+
+  "spell.rogues.shadow_step.name": "暗影步伐",
+  "spell.rogues.shadow_step.description": "穿过阴影出现在目标身后。",
+  "effect.rogues.shadow_step": "暗影步伐",
+  "effect.rogues.shadow_step.description": "不可追踪",
```

| **审查级别**    | 备注 |
|--------------|----|
| ⚪ 轻微 / 不阻止合并但需三思  |    | 
```suggestion
  "effect.rogues.shadow_step.description": "出其不意",
```
不可追踪可能更多形容类似“隐身”的效果。也可考虑贴合原文不改动。

---

感谢建议，已修改。